test/build.test.js

```diff
@@ -1,8 +1,9 @@
 const fs = require("fs");
 const Number = require("../lib/number");
 const pandoc = require("../lib/pandoc");
+const puppeteer = require("puppeteer");
 const assert = require("assert");
-const {PassThrough} = require("stream");
+const { PassThrough } = require("stream");
```

Comment:
Should we use [Prettier](https://prettier.io/) to automatically format our code?

Reply:
The `test/*.js`: yes, but the `test/test-data/*`: no.

Reply:
Then let's add a `.prettierignore` file, see https://prettier.io/docs/en/ignore.html#ignoring-files-prettierignore